internal/faustjs.org/docs/getting-started.mdx

```diff
@@ -71,6 +71,20 @@ Additionally, update the `FAUSTWP_SECRET_KEY` value with the secret key found in
   alt="The FaustWP WordPress admin interface with a red rectangle around the Secret Key field"
 />
 
+## Set your Front-end Site URL
+
+To ensure certain features of Faust function as expected, such as previews, be sure to set the Front-end Site URL in the Faust WordPress Plugin Settings. It is the URL to your headless front-end. This is used for authenticated post previews and for rewriting links to point to your front-end site.
+
+1. Navigate to the `Faust WordPress Plugin` settings page
+2. Enter in the base url of your headless WordPress site, i.e. http://localhost:3000 or http://www.mysiteurl.com in the Front-end site URL setting
+
+<img
+  src="/docs/img/frontend-site-url-settings.png"
```

Comment:
For other reviewers - this resolves to https://faustjs.org/docs/img/frontend-site-url-settings.png